<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H3 Setup Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="H3.qrc">
    <normaloff>:/images/H3.png</normaloff>:/images/H3.png</iconset>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Welcome to H3</string>
   </property>
   <property name="subTitle">
    <string>This wizard will guide you through the initial setup phase.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="infoLabel">
      <property name="text">
       <string>Please make sure you have the setup information provided by your focal point.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="infoLabel2">
      <property name="text">
       <string>You will need an active internet connection to complete this setup.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="ServerWizardPage" name="wizardPage2">
   <property name="title">
    <string>Database / Server setup</string>
   </property>
   <property name="subTitle">
    <string>Contact your focal point if needed.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="localDBlabel">
      <property name="text">
       <string>&amp;Local DB File :</string>
      </property>
      <property name="buddy">
       <cstring>localAddress</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="remoteDBlabel">
      <property name="text">
       <string>&amp;Server Address :</string>
      </property>
      <property name="buddy">
       <cstring>remoteAddress</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="localAddress">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>1.txt</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="remoteAddress">
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="localDBstatus">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="remoteDBstatus">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacerLocal">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacerRemote">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="LoginWizardPage" name="wizardPage3">
   <property name="title">
    <string>User setup</string>
   </property>
   <property name="subTitle">
    <string>Your username is provided by your focal point.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="2">
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacerUsername">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>&amp;Password :</string>
      </property>
      <property name="buddy">
       <cstring>passwordLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="confirmPasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="text">
       <string>secret</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="confirmPasswordLabel">
      <property name="text">
       <string>&amp;Confirm Password :</string>
      </property>
      <property name="buddy">
       <cstring>confirmPasswordLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>&amp;Username :</string>
      </property>
      <property name="buddy">
       <cstring>usernameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="text">
       <string>root</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLabel" name="passwordStatusLabel">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLabel" name="userStatusLabel">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacerPassword">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="RecapWizardPage" name="wizardPage4">
   <property name="title">
    <string>Setup complete !</string>
   </property>
   <property name="subTitle">
    <string>When you press finish, you will be ready to use H3</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QLabel" name="localRecap">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="localLabel">
      <property name="text">
       <string>Local DB :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="remoteLabel">
      <property name="text">
       <string>Remote DB :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="remoteRecap">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="userLabel">
      <property name="text">
       <string>User details :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="jobRecap">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="nameRecap">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="baseRecap">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="userActionRecap">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ServerWizardPage</class>
   <extends>QWizardPage</extends>
   <header>serverwizardpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoginWizardPage</class>
   <extends>QWizardPage</extends>
   <header>loginwizardpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecapWizardPage</class>
   <extends>QWizardPage</extends>
   <header>recapwizardpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>browseButton</tabstop>
  <tabstop>localAddress</tabstop>
  <tabstop>remoteAddress</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>confirmPasswordLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="H3.qrc"/>
 </resources>
 <connections/>
</ui>
